src/main/kotlin/io/github/mojira/arisa/infrastructure/ProjectCache.kt

```diff
@@ -0,0 +1,40 @@
+package io.github.mojira.arisa.infrastructure
+
+import io.github.mojira.arisa.jiraClient
+import java.time.Duration
+import java.time.Instant
+import net.rcarz.jiraclient.Project as JiraProject
+
+object ProjectCache {
+    private var cache = Cache<JiraProject>()
+    private var lastRefresh = Instant.now()
+
+    private const val REFRESH_INTERVAL_IN_MINUTES: Long = 5
```

Comment:
We have the command to clear it and versions are created much earlier than they are actually released, how about we set it to an hour?

Reply:
The thing is, the version info includes stuff like "is this version released or not", which e.g. FutureModule relies on. With 5 minutes, the delay would be barely noticeable if nobody runs the command, while with 60 minutes effectively we're requiring someone to run the command during the first hour after version release.

I don't think the call should be _too_ expensive to do once every five minutes. Currently we're calling it every time a ticket is updated, so once every second or so.

If a call every five minutes is still too expensive, we can still up this number easiliy in a future PR.